### ISSUE TITLE
False Ban notice, claims i sent false or blank data

### ISSUE THREAD

**Issue by Cuboos** [tgstation/-tg-station#9325]
Banned by host: Your computer provided invalid or blank information to the server on connection (byond username, IP, and Computer ID.) Provided information for reference: Username:'' IP:'blank' Computer ID:'' If you continue to get this error, please restart byond or contact byond support.
Access denied.

I got this message and got disconnected near the end of a round. However, i was able to reconnect and continue playing.

**Comment by Jordie0608** [comment]
This is a problem with BYOND; nothing we can do.

**Comment by Incoming5643** [comment]
It'd be nice if this specific message had a less harsh phrasing though, as it's far more likely to appear as a result of byond issues than as malicious spoofing.

**Comment by WJohn** [comment]
Yeah I've had this happen to me many many times for the past few months, but never thought to report it.

**Comment by MrStonedOne** [comment]
You can't fix the phrasing. the banned by host bit is added by byond.

**Comment by MrStonedOne** [comment]
Mind you, getting rejected for being a guest says "banned by host"

Getting rejected for connection because of the extreme pop cap says "banned by host"

**Comment by astralenigma** [comment]
Weirdly I'm seeing this bug too but it's always at the end of the round and it has been only quite recently isn't it possible that some code we made recently might lead BYOND to send that message? Because until recently I hadn't seen that message after a round end, and it also happened in a server hosted by a friend of mine and he didn't have more than 20 players.

**Comment by MrStonedOne** [comment]
This error message is generated by the game, not byond. It comes up when byond didn't send the persons info to the ban checker. This is a direct hook into byond with no intermediate code. So its a byond bug. 

A month or so ago I noticed that this could be exploited to ban evade by getting it to trigger manually. So I made it deny connections with out the proper info and filed a bug report with byond. The "banned by host" part of the message is added by byond, can't change that, but the rest of the message is us
